### PR TITLE
NXP-31018: preview compound document

### PR DESCRIPTION
**Pull request by richardsd**
Feedback is very welcome.
~~Two solutions (each per commit) to fetch the document (properties) and display it in the `nuxeo-document-previewer`.~~ 
- Going with the second solution, it is cleaner and similar to strategies used in other packages/addons.
- New: Added functional test for NXP-31019

### CODE REVIEW

**Review by Gabez0r — COMMENTED** [review]
Good job! I think I like the second approach better. btw.

**Comment by richardsd** [comment]
> Good job! I think I like the second approach better. btw.

Same. I reused the strategy we're using in the 3d addon 🙂

**Review by Gabez0r — DISMISSED** [review]

**Review by semisse — DISMISSED** [review]

File: nuxeo-compound-documents-web/ftest/features/preview.feature [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Feature: Check the preview of a Compound Document
+
+    I can see the preview of a Compound Document
+
+  Background:
+    Given user "John" exists in group "members"
+    And I login as "John"
+    And I have a Workspace document
+    And I have permission ReadWrite for this document
+    And I browse to the document
```
Comment (Gabez0r) [inline nuxeo-compound-documents-web/ftest/features/preview.feature]: I don't think we need this step?

File: nuxeo-compound-documents-web/ftest/features/preview.feature [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    And I have a CompoundDocument imported from file "samplecompound.zip"
+    And I have permission Read for this document
+    And I browse to the document with path "/default-domain/my_document"
```
Comment (Gabez0r) [inline nuxeo-compound-documents-web/ftest/features/preview.feature]: Do we need to browse and then browse again in the next step?

File: nuxeo-compound-documents-web/ftest/features/preview.feature [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+  Scenario: Check the preview of a compound document
```
Comment (Gabez0r) [inline nuxeo-compound-documents-web/ftest/features/preview.feature]: If we just have a single scenario, I think we don't need the background.

**Review by Gabez0r — COMMENTED** [review]
Great job! 💪

File: nuxeo-compound-documents-web/ftest/features/preview.feature (hunk 2 above)
Comment (richardsd) [reply nuxeo-compound-documents-web/ftest/features/preview.feature]: Ups, I think I made a wrong copy paste 😬 Let me review it.

File: nuxeo-compound-documents-web/ftest/features/preview.feature (hunk 1 above)
Comment (richardsd) [reply nuxeo-compound-documents-web/ftest/features/preview.feature]: ~~I need to confirm.~~ Correct, I'm removing it.

**Comment by richardsd** [comment]
@Gabez0r @semisse asking for another round of reviews 🙏 Thanks

**Review by semisse — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]
LGTM! 💪

**Pull request merged**